dae-alright/src/components/search-bar/component/button/button.component.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+// import './button.style.scss';
```

Comment:
Please remove this

Reply:
@samynaj why haven't made resolved this?